src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,34 @@
+.catalog {
+  &__wrapper {
```

Comment:
Why did you add extra wrapper? You can wrote all this styles inside .catalog

Reply:
Agree, this wrapper can be removed

Reply:
i need this wrapper, because i set margin, and i cant set it for block

Reply:
Unnecessary wrapper.